src/fabric.c

```diff
@@ -437,6 +437,7 @@ void fi_ini(void)
 	ofi_register_provider(VERBS_INIT, NULL);
 	ofi_register_provider(GNI_INIT, NULL);
 	ofi_register_provider(BGQ_INIT, NULL);
+	ofi_register_provider(NETDIR_INIT, NULL);
```

Comment:
Doesn't adding this here break the windows build?  There is no fi_netdir_ini() function included as part of the source yet.